## 5. Numeric Field Not Entered as Numeric

Fields can be either alphanumeric or numeric. This error is flagged
when the field is expected to be a number and it is not.

Example of *incorrect* code:

```
Timestep,
    Not a Number;
```

